File: Punto2/estudiante.ts
interface Estudiante {
    nombre: string;
    edad: number;
    promedio: number;
}

const estudiantes: Estudiante[] = [
    { nombre: "Carlos Gomez", edad: 20, promedio: 4.5 },
    { nombre: "Maria Rodriguez", edad: 22, promedio: 4.8 },
    { nombre: "Juan Perez", edad: 19, promedio: 3.9 }
];

estudiantes.forEach(e => {
    console.log(`Nombre: ${e.nombre}, Edad: ${e.edad}, Promedio: ${e.promedio}`);
});

File: Punto2/estudiante.dart
class Estudiante {
  final String nombre;
  final int edad;
  final double promedio;

  Estudiante(this.nombre, this.edad, this.promedio);
}

void main() {
  final estudiantes = [
    Estudiante("Carlos Gomez", 20, 4.5),
    Estudiante("Maria Rodriguez", 22, 4.8),
    Estudiante("Juan Perez", 19, 3.9),
  ];

  for (var e in estudiantes) {
    print("Nombre: ${e.nombre}, Edad: ${e.edad}, Promedio: ${e.promedio}");
  }
}
